src/codegen/templates/property_setter_template.py.mako

```diff
@@ -43,15 +44,22 @@
             function_call_args.append("\"\"")
         function_call_args.append(hex(attribute.id))
         function_call_args.append('val')
-        if attribute.name in ACTIVE_DEVS_SUPPORTED_ATTRIBUTES:
+        if attribute.name in active_devs_supported_attributes:
             function_call_args_ex = function_call_args.copy()
             function_call_args_ex.insert(1, "self._active_devs")
     %>\
-    %if attribute.name in ACTIVE_DEVS_SUPPORTED_ATTRIBUTES:
+    %if attribute.python_class_name == "Timing":
+        %if attribute.name in active_devs_supported_attributes:
         if self._active_devs:
             self._interpreter.set_${generic_attribute_func_ex}(${', '.join(function_call_args_ex)})
         else:
             self._interpreter.set_${generic_attribute_func}(${', '.join(function_call_args)})
+        %else:
+        if self._active_devs == None:
```

Comment:
[style] Be consistent about using falsiness (`if self._active_devs:`) or explicitly comparing to None. Also, if you are going to explicitly compare to None, use `is` or `not is`, not `==` or `!=`.